### PR TITLE
adding Graphic Equalizer and ColorPicker Library

### PR DESCRIPTION
**Pull request by jposada202020**
Thanks

### CODE REVIEW

File: circuitpython_community_library_list.md [hunk 1]
```diff
@@ -60,11 +60,13 @@ Here is a listing of current CircuitPython Community Libraries. These libraries
 * [CircuitPython AzureCustomVision Prediction](https://github.com/jimbobbennett/CircuitPython_AzureCustomVision_Prediction) ([PyPi](https://pypi.org/project/circuitpython-azurecustomvision-prediction)) \([Docs](https://circuitpython.readthedocs.io/projects/circuitpython_azurecustomvision_prediction/en/latest/))
 * [CircuitPython Base64](https://github.com/jimbobbennett/CircuitPython_Base64) ([PyPi](https://pypi.org/project/circuitpython-base64)) \([Docs](https://circuitpython.readthedocs.io/projects/base64/en/latest/))
 * [CircuitPython Candlesticks](https://github.com/jposada202020/CircuitPython_Candlesticks.git) ([PyPi](https://pypi.org/project/circuitpython-Candlesticks)) \([Docs](https://circuitpython-candlesticks.readthedocs.io/))
+* [CircuitPython ColorPicker](https://github.com/jposada202020/CircuitPython_color_picker.git) ([PyPi] \([Docs](https://circuitpython-color-picker.readthedocs.io/))
```
Comment (tannewt) [inline circuitpython_community_library_list.md:63]: Want to add the PyPi link here?

Comment (jposada202020) [reply circuitpython_community_library_list.md]: Added :)

**Review by tekktrik — APPROVED** [review]
Looks awesome!

**Comment by jposada202020** [comment]
Thank you :)

**Pull request merged**